package-lock.json

```diff
@@ -2,11 +2,821 @@
   "requires": true,
```

Comment:
looks like you installed a bunch of dependencies from your local machine maybe? 

I think we want the old `package-lock.json` here. 

Try deleting this file from your local and running `npm install` to fix